README.md

```diff
@@ -50,7 +50,7 @@ const x = await import("<specifier>", { phase: "source" });
 ```
 
 For dynamic imports, import phase is specified in the same second
-attribute options bag that [import assertions][] are specified in, using the
+attribute options bag that [import attributes][] are specified in, using the
 `phase` key.
```

Comment:
It is not immediately clear from this sentence that `phase` is not _within_ the attributes bag. Maybe rephrase or add an example?

Reply:
There is an example before, but I've updated the wording to be clearer.